vsintegration/Vsix/VisualFSharpOpenSource/VisualFSharpOpenSource.csproj

```diff
@@ -324,8 +324,8 @@
       <SpecificVersion>False</SpecificVersion>
       <HintPath>$(FSharpSourcesRoot)\..\packages\Newtonsoft.Json.10.0.2\lib\net45\Newtonsoft.Json.dll</HintPath>
     </Reference>
-    <Reference Include="System.Collections.Immutable, Version=1.3.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
-      <HintPath>$(FSharpSourcesRoot)\..\packages\System.Collections.Immutable.$(SystemCollectionsImmutableVersion)\lib\portable-net45+win8+wp8+wpa81\System.Collections.Immutable.dll</HintPath>
+    <Reference Include="System.Collections.Immutable, Version=1.2.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
```

Comment:
Is assembly version 1.3.1 really in package 1.2.1?  gosh that seems idiosyncratic.

Reply:
Yes

Reply:
Discovered that the hard way

Reply:
But hey, think what people say about FSharp.Core...

Reply:
Well .. that sure is a thing ...